### PR TITLE
Add additional RPC info

### PR DESCRIPTION
**Pull request by nadinequinr3**

### CODE REVIEW

File: content/en/platform/corda/5.0-alpha/developing/rest-api/rest-api.md [hunk 1]
```diff
@@ -8,4 +9,21 @@ menu:
...
+1. To access the RPC endpoint of a Corda cluster, forward the port by running the following command in a terminal window:
+
+   ```sh
+   kubectl port-forward -n corda deploy/corda-rpc-worker 8888
+   ```
+
+   You can then access the Swagger documentation for the RPC endpoint at [https://localhost:8888/api/v1/swagger](https://localhost:8888/api/v1/swagger).
+Note that the RPC endpoint is protected by a self-signed certificate.
+
+2. To invoke the RPC endpoint from the Swagger UI, use the username `admin` and password `admin`. Alternatively, you can use curl. For example:
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.0-alpha/developing/rest-api/rest-api.md:23]:
```suggestion
2. To invoke the RPC endpoint from the Swagger UI, use the user name `admin` and password `admin`. Alternatively, you can use curl. For example:
```

File: content/en/platform/corda/5.0-dev-preview-2/developing/rest-api/rest-api.md [hunk 2]
```diff
@@ -8,4 +8,21 @@ menu:
...
+1. To access the RPC endpoint of a Corda cluster, forward the port by running the following command in a terminal window:
+
+   ```sh
+   kubectl port-forward -n corda deploy/corda-rpc-worker 8888
+   ```
+
+   You can then access the Swagger documentation for the RPC endpoint at [https://localhost:8888/api/v1/swagger](https://localhost:8888/api/v1/swagger).
+Note that the RPC endpoint is protected by a self-signed certificate.
+
+2. To invoke the RPC endpoint from the Swagger UI, use the username `admin` and password `admin`. Alternatively, you can use curl. For example:
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.0-dev-preview-2/developing/rest-api/rest-api.md:22]:
```suggestion
2. To invoke the RPC endpoint from the Swagger UI, use the user name `admin` and password `admin`. Alternatively, you can use curl. For example:
```

File: content/en/platform/corda/5.0-dev-preview-2/developing/rest-api/rest-api.md [hunk 3]
```diff
@@ -8,4 +8,21 @@ menu:
...
+To access and invoke the REST API:
+
+1. To access the RPC endpoint of a Corda cluster, forward the port by running the following command in a terminal window:
+
+   ```sh
+   kubectl port-forward -n corda deploy/corda-rpc-worker 8888
+   ```
+
+   You can then access the Swagger documentation for the RPC endpoint at [https://localhost:8888/api/v1/swagger](https://localhost:8888/api/v1/swagger).
+Note that the RPC endpoint is protected by a self-signed certificate.
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.0-dev-preview-2/developing/rest-api/rest-api.md:20]: Add note code snippets?

File: content/en/platform/corda/5.0-alpha/developing/rest-api/rest-api.md [hunk 4]
```diff
@@ -8,4 +9,21 @@ menu:
...
+To access and invoke the REST API:
+
+1. To access the RPC endpoint of a Corda cluster, forward the port by running the following command in a terminal window:
+
+   ```sh
+   kubectl port-forward -n corda deploy/corda-rpc-worker 8888
+   ```
+
+   You can then access the Swagger documentation for the RPC endpoint at [https://localhost:8888/api/v1/swagger](https://localhost:8888/api/v1/swagger).
+Note that the RPC endpoint is protected by a self-signed certificate.
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.0-alpha/developing/rest-api/rest-api.md:21]: Add note code snippets?

**Review by AnnaK-R3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
thanks @nadinequinr3

**Pull request merged**